luxonis_train/nodes/heads/efficient_bbox_head.py

```diff
@@ -27,32 +25,30 @@ class EfficientBBoxHead(
 
     def __init__(
         self,
-        n_heads: Literal[2, 3, 4] = 3,
+        num_heads: Literal[2, 3, 4] = 3,
         conf_thres: float = 0.25,
         iou_thres: float = 0.45,
         max_det: int = 300,
-        **kwargs,
+        **kwargs: Any,
     ):
         """Head for object detection.
 
-        TODO: add more documentation
-
-        @type n_heads: Literal[2,3,4]
-        @param n_heads: Number of output heads. Defaults to 3.
-          ***Note:*** Should be same also on neck in most cases.
-
+        Adapted from U{YOLOv6: A Single-Stage Object Detection Framework for Industrial
+        Applications <https://arxiv.org/pdf/2209.02976.pdf>}.
+        @type num_heads: Literal[2,3,4]
+        @param num_heads: Number of output heads. Defaults to 3. B{Note:} Should be same
+            also on neck in most cases.
         @type conf_thres: float
         @param conf_thres: Threshold for confidence. Defaults to C{0.25}.
-
         @type iou_thres: float
         @param iou_thres: Threshold for IoU. Defaults to C{0.45}.
-
         @type max_det: int
-        @param max_det: Maximum number of detections retained after NMS. Defaults to C{300}.
+        @param max_det: Maximum number of detections retained after NMS. Defaults to
+            C{300}.
         """
         super().__init__(**kwargs)
 
-        self.n_heads = n_heads
+        self.num_heads = num_heads
```

Comment:
Should we rather stick with `n_heads` as we have `n_classes`, `n_keypoints`,... Same question for other heads (efficient_keypoint and implicit_keypoint), modules and docks where `num_heads` appears

Reply:
Should we rename all the `num_something` to `n_something` to be consistent? Like `n_workers`, `n_logged_images` etc.?

Reply:
IMO we should yeah